src/main/java/io/github/sskorol/assertions/CustomAssertions.java

```diff
@@ -12,11 +12,11 @@ private CustomAssertions() {
         throw new UnsupportedOperationException("Illegal access to private constructor.");
     }
 
-    public static LoginPageAssert customAssertThat(final Account account) {
-        return new ModelSoftAssert().assertThat(LoginPageAssert.class, Account.class, account);
+    public static LoginAssert customAssertThat(final LoginPage loginPage) {
```

Comment:
I think you will change your custom assert name to more obvious. For example:
`LoginPageAssert`